index.js

```diff
@@ -46,7 +46,32 @@ module.exports = function(app) {
       interval: {
         type: 'number',
         title: 'Log rotation interval (in seconds). Value of zero disables log rotation.',
-        default: 600
+        default: 600 // should this default be hourly (3600) rather than every 10 min?
```

Comment:
I don't have a strong opinion here. There's less uncompressed data the shorter the interval is, but on the other hand, the number of compressed log files grows quite large. Might as well be 3600 if you think that's preferable. Would you mind removing the comment and then setting the value to either one? :-)